db/migrate/20151215172759_add_status_and_paragraph_number_to_references.rb

```diff
@@ -0,0 +1,6 @@
+class AddStatusAndParagraphNumberToReferences < ActiveRecord::Migration
+  def change
+    add_column :references, :status, :string, default: "pending"
```

Comment:
C'est bien cela => default: pending
La référence est créée et sauvée dans la db au moment de l'upload, avec statut pending. Ce n'est qu'en cas de confirmation de l'utilisateur que le status = up-to-date